account-kit/react/src/components/auth/hooks/useOAuthVerify.ts

```diff
@@ -0,0 +1,49 @@
+"use client";
+import { useCallback } from "react";
+import { useAuthenticate } from "../../../hooks/useAuthenticate.js";
+import { useAuthContext } from "../context.js";
+import { useUiConfig } from "../../../hooks/useUiConfig.js";
+
+export type UseOAuthVerifyReturnType = {
+  authenticate: () => void;
+  isPending: boolean;
+};
+
+export const useOAuthVerify = (): UseOAuthVerifyReturnType => {
+  const { setAuthStep } = useAuthContext();
+  const { supportUrl } = useUiConfig();
```

Comment:
we probs don't need to drill this down into the `authStep`, we can just use this hook when we need it